prime-dotnet-webapi/Services/interfaces/ISiteService.cs

```diff
@@ -13,6 +13,7 @@ public interface ISiteService
         Task<bool> SiteExistsAsync(int siteId);
         Task<SiteStatusType> GetSiteCurrentStatusAsync(int siteId);
         Task<bool> PecAssignableAsync(int siteId, string pec);
+        Task<bool> PecExistsWithinHA(int siteId, string pec);
```

Comment:
Convention is to suffix these type of method names' with `Async`

Reply:
Fixed

Reply:
Fixed